app/views/foreman/job_templates/update_packages_by_search_query_-_katello_ansible_default.erb

```diff
@@ -3,7 +3,7 @@ kind: job_template
 name: Update packages by search query - Katello Ansible Default
 job_category: Katello
```

Comment:
What is a bit funny it that this template has the category `Katello` and the template `Update Package - Katello Ansible Default` has the category `Katello via Ansible`.

This results in a separate section for Katello jobs with the caption Ansible:
![image](https://github.com/Katello/katello/assets/71487468/759e43fe-d018-4341-b6f4-76ac16c1efc5)

IMHO all of them should go to `Katello via Ansible`, otherwise this is somehow inconsistent (but this is out of scope of this PR).

Reply:
I have created a separate issue: https://projects.theforeman.org/issues/37362